bin/gbuild

```diff
@@ -329,10 +329,16 @@ build_desc["remotes"].each do |remote|
     if @options[:fetch_branches]
       system!("cd inputs/#{dir} && git fetch -f --update-head-ok #{sanitize_path(remote["url"], remote["url"])} +refs/heads/*:refs/heads/*")
     end
+    commit_fetch = commit
     if @options[:fetch_tags]
       system!("cd inputs/#{dir} && git fetch -f --update-head-ok #{sanitize_path(remote["url"], remote["url"])} +refs/tags/*:refs/tags/*")
+    else
+      refinfo = `cd inputs/#{dir} && git ls-remote #{sanitize_path(remote["url"], remote["url"])} #{commit}`
+      if refinfo.include? "\trefs/tags/"
+        commit_fetch = "tag " + commit
```

Comment:
I don't quite understand this.  it would result in the command below being something like `git fetch ... tag xxx`, but that's not a valid way to supply args to `git fetch`?  if you believe it's correct, can you give me an example of what the `git fetch` command line will look like?

Reply:
did you mean `--tag`?

Reply:
From the manpage:

```
           tag <tag> means the same as refs/tags/<tag>:refs/tags/<tag>; it requests fetching everything up to the given tag.
```

Reply:
I understand, was missing some git knowledge